cmd/nvidia-cdi-hook/create-symlinks/create-symlinks.go

```diff
@@ -211,21 +211,14 @@ func changeRoot(current string, new string, path string) (string, error) {
 // Locate returns the link target of the specified filename or an empty slice if the
 // specified filename is not a symlink.
 func (m command) Locate(filename string) ([]string, error) {
-	info, err := os.Lstat(filename)
+	target, err := symlinks.Resolve(filename)
 	if err != nil {
-		return nil, fmt.Errorf("failed to get file info: %v", info)
+		return nil, err
```

Comment:
Why not return a wrapped error that mentions the symlink name?

Reply:
I was trying to reduce the verbosity here. This is a low-level function and the caller can decide on whether to log / output more context.